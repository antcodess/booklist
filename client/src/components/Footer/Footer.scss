.footer {
  background-color: #F4F2F2;
  margin-top: 130px;

  .footer-content {
    margin: 0 auto;
    text-align: center;
    padding: 0px 100px;
    max-width: 1200px;

    .footer-top {
      padding-top: 70px;
      display: flex;
      justify-content: space-between;
      align-items: center;

      h1 {
        font-size: 2.5rem;
      }
      
      .footer-list {
        display: flex;
        justify-content: space-between;

        li {
          margin: 0 5px;
          border-left: black 1px solid;
          padding-left: 10px;
          cursor: pointer;

          &:hover {
            opacity: 0.6;
          }
        }

        li.first {
          border-left: none;
        }

        li.last {
          margin-right: 0;
        }
      }
    }

    .footer-bottom {
      display: flex;
      border-top: 1px #C3C3C3 solid;
      margin-top: 50px;
      padding-bottom: 70px;

      .footer-bottom-text {
        margin-top: 20px;
      }
    }
  }
}

@media only screen and (max-width: 830px) {
  .footer-top {
    flex-direction: column;
  }

  .footer-bottom {
    justify-content: center;
  }

  .footer-top h1 {
    margin-bottom: 30px;
  }
}

@media only screen and (max-width: 670px) {
  .footer {
    .footer-content {
      .footer-top {
        .footer-list {
          flex-direction: column;
          li {
            border-left: none;
          }
        }
      }
    }
  }
}

@media only screen and (max-width: 500px) {
  .footer {
    .footer-content {
      .footer-top {
        .footer-list {
          flex-direction: column;
          li {
            margin-bottom: 10px;
            padding-left: 0;
          }
        }
      }
    }
  }
}

@media only screen and (max-width: 480px) {
  .footer {
    .footer-content {
      .footer-bottom {
        .footer-bottom-text {
          text-align: center;
        }
      }
    }
  }
}