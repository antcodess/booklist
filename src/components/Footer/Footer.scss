.footer {
  background-color: #F4F2F2;
  margin-top: 200px;

  .footer-content {
    margin: 0 auto;
    text-align: center;
    padding: 0px 100px;
    max-width: 1720px;

    .footer-top {
      padding-top: 70px;
      display: flex;
      justify-content: space-between;
      align-items: center;

      h1 {
        font-size: 2.5rem;
      }
      
      .footer-list {
        display: flex;
        // flex-wrap: wrap;
        justify-content: space-between;

        li {
          margin: 0 5px;
          border-left: black 1px solid;
          padding-left: 10px;
          cursor: pointer;

          &:hover {
            opacity: 0.6;
          }
        }

        li.first {
          border-left: none;
        }

        li.last {
          margin-right: 0;
        }
      }
    }

    .footer-bottom {
      display: flex;
      border-top: 1px #C3C3C3 solid;
      margin-top: 50px;
      padding-bottom: 70px;

      .footer-bottom-text {
        margin-top: 20px;
      }
    }
  }

}