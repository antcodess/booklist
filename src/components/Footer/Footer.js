import React from 'react';

import './Footer.scss';

const Footer = () => {
  return (
    <footer className='footer'>
      <div className='footer-content'>
        <div className='footer-top'>
          <h1>BookList</h1>
          <div className='footer-nav'>
            <ul className='footer-list'>
              <li className='first'>About</li>
              <li>Careers</li>
              <li>Conditions of Use</li>
              <li>Privacy Notice</li>
              <li className='last'>Contact</li>
            </ul>
          </div>
        </div>
        <div className='footer-bottom'>
          <span className='footer-bottom-text'>&copy; Copyright 1997-2019 BookList, Inc</span>
        </div>
      </div>
    </footer>
  );
};

export default Footer;