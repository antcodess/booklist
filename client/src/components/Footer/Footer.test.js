import React from 'react';
import { shallow } from 'enzyme';
import { Footer } from './Footer';

describe('Footer Component', () => {
  let wrapper;
  
  beforeEach(() => {
    wrapper = shallow(<Footer />);
  });

  it('should render Footer Component', () => {
    expect(wrapper).toMatchSnapshot();
  });
});