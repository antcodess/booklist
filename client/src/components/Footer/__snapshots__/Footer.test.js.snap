// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component should render Footer Component 1`] = `ShallowWrapper {}`;
